digiwf-apps/packages/apps/digiwf-tasklist/src/middleware/tasks/taskMiddleware.ts

```diff
@@ -60,10 +61,10 @@ const addUserToTask = (r: Task): Promise<HumanTask> => {
 };
 
 const handlePageOfTaskResponse = (response: PageOfTasks) => {
-  const tasksWithUserPromises = response.content?.map(addUserToTask);
+  const tasksWithUserPromises = response.content?.map(addUserToTask) || [];
   return Promise.all<HumanTask>(tasksWithUserPromises)
     .then(tasks => Promise.resolve<Page<HumanTask>>({
-        content: tasks,
+        content: removeFinishedTasks(tasks),
         totalElements: response.totalElements,
         totalPages: response.totalPages!,
```

Comment:
Das passt dann nicht mit der Liste überein 🙈

Reply:
Hallo, 
ich verstehe den Kommentar nicht. 
Meine Hypothese ist: Du meinst, dass wenn wir tasks rausfiltern, dass die Anzahl der Tasks und ggf. die Anzahl an Seiten nicht passt (da ggf. zu hoch). 
Für den Fall das meine Hypothese stimmt: 
Im Normalfall reden wir von einer fehlerhaften Angabe der Anzahl der Tasks sowie ggf. Anzahl der Seiten von mehreren Sekunden. Sobald der Nutzer wieder einen Task bearbeitet, wird die Liste im Hintergrund aktualisiert. 

Aus meiner Sicht wäre das ein Fortschritt für die Testphase. Ich kann aber auch gern mehr Zeit investieren und clientseitig die Anzahl der Gesamtelemente sowie die Anzahl der Seiten neu berechnen.

Reply:
Ich denke, dass wir das hier einfach und transparent lösen können. Daher favorisiere ich hier Lösung 2:

https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/issues/gh/it-at-m/digiwf-core/400#issuecomment-1576542527